ext/Draw/CDraw3DLine.cpp

```diff
@@ -23,16 +23,22 @@ CDraw3DLine::CDraw3DLine(IVideoDriver* _driver)
         Draw3DLineVertexShader,
         nullptr,nullptr,nullptr,
         Draw3DLineFragmentShader,
-        3,EMT_SOLID,
+        2,EMT_SOLID,
         callBack,
         0);
     callBack->drop();
+
+    m_lineData[1] = (void*) m_indices;
+    m_meshBuffer->setPrimitiveType(EPT_LINES);
+    m_meshBuffer->setMeshDataAndFormat(m_desc);
+    m_desc->drop();
+    m_meshBuffer->setIndexType(EIT_16BIT);
```

Comment:
if getting rid of indices then set this to EIT_UNKNOWN

Reply:
commiting this